setup.cfg

```
@@ -32,6 +32,7 @@ project_urls =
     Documentation = https://django-auth-ldap.readthedocs.io/
     Source = https://github.com/django-auth-ldap/django-auth-ldap
     Tracker = https://github.com/django-auth-ldap/django-auth-ldap/issues
+    Changelog = https://django-auth-ldap.readthedocs.io/en/latest/changes.html
```

GitHub releases have been used for the recent versions.

```suggestion
    Changelog = https://github.com/django-auth-ldap/django-auth-ldap/releases/
```